plugins/query-insights/src/main/java/org/opensearch/plugin/insights/core/exporter/QueryInsightsExporterType.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.insights.core.exporter;
+
+import java.util.Locale;
+
+/**
+ * Types for the Query Insights Exporters
+ *
+ * @opensearch.internal
+ */
+public enum QueryInsightsExporterType {
+    /* local index exporter */
+    LOCAL_INDEX("local_index");
+
+    private final String type;
+
+    QueryInsightsExporterType(String type) {
+        this.type = type;
+    }
+
+    public static QueryInsightsExporterType parse(String type) {
+        return valueOf(type.toUpperCase(Locale.ROOT));
+    }
```

Comment:
The `type` field on the enum instance is not doing anything here.

Reply:
Removed the type field in this enum.